play-media-native/screens/CreateMedia.tsx

```diff
@@ -96,18 +117,34 @@ export const CreateMediaScreen = ({ navigation, route }) => {
     }, [route.params.image])
   );
 
+  useEffect(() => {
+    // init global state on mount
+    //
+    if (stateKey) {
+      init({
```

Comment:
Is there a reason for initializing this? Just asking because all changes on this screen are local.

Reply:
Yes, in order to initialize the content fields and use them later to keep the information after dismissing the discard screen and continuing the creation flow.

Reply:
The info was still present previously when coming back from the discard changes screen, but was just overwritten by this initialization   
![image](https://user-images.githubusercontent.com/96472119/230359117-29718604-2633-4142-944e-1c0029863706.png)